main.go

```diff
@@ -77,3 +78,11 @@ func getEnvDefault(key string, defaultVal string) string {
 
 	return val
 }
+
+func getURLUsingPlek(key string) string {
+	url, err := plek.Find(key)
+	if err != nil {
+		log.Fatal(err)
+	}
+	return url.String()
```

Comment:
It just seemed like a sensible approach, given Go provides a type for holding a URL.  It would avoid us having to do URL parsing deep within the application.

Having said that, after digging further, `http.NewRequest` takes a string, not a `url.URL`, so that negates any benefit of doing this.

I therefore wonder whether it would be better for us to have Plek return strings as opposed to `url.URL` instances.  Thoughts?  Given this is the first thing using the Go version of Plek, now is the time to get this right...